src/layers/selayers.jl

```diff
@@ -21,19 +21,13 @@ Creates a squeeze-and-excitation layer used in MobileNets, EfficientNets and SE-
   - `norm_layer`: The normalization layer to be used after the convolution layers
   - `rd_planes`: The number of hidden feature maps in a squeeze and excite layer
 """
-# TODO look into a `get_norm_act` layer that will return a closure over the norm layer
-# with the activation function passed in when the norm layer is not `identity`
-function squeeze_excite(inplanes::Integer, squeeze_planes::Integer;
-                        norm_layer = planes -> identity, activation = relu,
-                        gate_activation = sigmoid)
-    layers = [AdaptiveMeanPool((1, 1)),
-        Conv((1, 1), inplanes => squeeze_planes),
-        norm_layer(squeeze_planes),
-        activation,
-        Conv((1, 1), squeeze_planes => inplanes),
-        norm_layer(inplanes),
-        gate_activation]
-    return SkipConnection(Chain(filter!(!=(identity), layers)...), .*)
+function squeeze_excite(inplanes::Integer, squeeze_planes::Integer; norm_layer = identity,
+                        activation = relu, gate_activation = sigmoid)
+    return SkipConnection(Chain(AdaptiveMeanPool((1, 1)),
+                                conv_norm((1, 1), inplanes, squeeze_planes, activation;
+                                          norm_layer)...,
+                                conv_norm((1, 1), squeeze_planes, inplanes,
+                                          gate_activation; norm_layer)...), .*)
```

Comment:
TIL that `(.*)` returns a `Base.Broadcast.BroadcastFunction`. Maybe we should create an alias for it in NNlib...